app/controllers/comments_controller.rb

```diff
@@ -1,10 +1,10 @@
 class CommentsController < ApplicationController
   def create
     @comment = current_user.comments.new(comment_params)
-    @post = Post.find(params[:comment][:post_id])
+    @post = Post.find(comment_params[:post_id])
 
     if @comment.save
-      redirect_to "#{users_path}/#{@post.author_id}/posts/#{@post.id}"
+      redirect_to user_post_path @post
     else
       render :new, status: :unprocessable_entity
```

Comment:
Please use flash messages in `posts#create`, `comments#create`, and `likes#create` actions for both successful and failed paths. You may learn more about how to use flash messages [here](https://medium.com/@nerdplusdog/back-in-a-flash-using-flash-messages-in-your-ruby-on-rails-apps-c39e09c15551).

Reply:
Totally forgot about this. Thanks for reminding me.